packages/node_modules/@webex/webex-widget-base/package.json

```diff
@@ -7,6 +7,10 @@
   "keywords": [],
   "author": "devsupport@webex.com",
   "license": "MIT",
+  "dependencies": {
```

Comment:
Unfortunately, these get overwritten by the build system. If you run `npm run build package.json` you'll see that it no longer has this dependency.

Whatever solution we come up with will need to work with or along side of the [deps.js](https://github.com/adamweeks/react-widgets/blob/e9f321b96216fee09aa0905ab03d29300a211b31/scripts/utils/deps.js#L19) script.

Reply:
I updated the deps.js script to look for existing dependencies first and if not just create a simple object

Reply:
that should fix the issue